examples/aks/stack.tf

```
@@ -0,0 +1,192 @@
+}
+
+module "azure-workload-identity" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-azure-workload-identity.git?ref=v0.1.0"
+
+  argocd_namespace = module.argocd.argocd_namespace
+
+  azure_tenant_id = data.azuread_client_config.current.tenant_id
+}
+
+module "cert-manager" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-cert-manager.git//aks?ref=v4.0.0"
+
+  cluster_name     = local.cluster_name
+  base_domain      = azurerm_dns_zone.this.name
+  argocd_namespace = module.argocd.argocd_namespace
+
+  cluster_oidc_issuer_url      = module.cluster.oidc_issuer_url
+  node_resource_group_name     = module.cluster.node_resource_group
+  dns_zone_resource_group_name = azurerm_resource_group.default.name
+  enable_service_monitor       = false
+
+  dependency_ids = {
+    azure-workload-identity = module.azure-workload-identity.id
+  }
+}
+
+module "loki-stack" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-loki-stack.git//aks?ref=v2.2.0"
+
+  argocd_namespace = module.argocd.argocd_namespace
+
+  distributed_mode = true
+
+  logs_storage = {
+    container                        = azurerm_storage_container.logs.name
+    storage_account                  = azurerm_storage_account.this.name
+    managed_identity_node_rg_name    = module.cluster.node_resource_group
+    managed_identity_oidc_issuer_url = module.cluster.oidc_issuer_url
+  }
+
+  dependency_ids = {
+    azure-workload-identity = module.azure-workload-identity.id
+  }
+}
+
+module "thanos" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-thanos.git//aks?ref=v1.0.0"
+
+  cluster_name     = local.cluster_name
+  base_domain      = azurerm_dns_zone.this.name
+  argocd_namespace = module.argocd.argocd_namespace
+  cluster_issuer   = "letsencrypt-staging"
+
+  metrics_storage = {
+    container           = azurerm_storage_container.metrics.name
+    storage_account     = azurerm_storage_account.this.name
+    storage_account_key = azurerm_storage_account.this.primary_access_key
+  }
+
+  thanos = {
+    oidc = local.oidc
+  }
+
+  dependency_ids = {
+    cert-manager = module.cert-manager.id
+  }
+}
+
+module "monitoring" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-kube-prometheus-stack.git//aks?ref=v2.3.0"
+
+  cluster_name     = local.cluster_name
+  base_domain      = azurerm_dns_zone.this.name
+  argocd_namespace = module.argocd.argocd_namespace
+  cluster_issuer   = "letsencrypt-staging"
+
+  alertmanager = {
+    oidc = local.oidc
+  }
+  prometheus = {
+    oidc = local.oidc
+  }
+  grafana = {
+    oidc                    = local.oidc
+    additional_data_sources = true
+  }
+
+  metrics_storage = {
+    container           = azurerm_storage_container.metrics.name
+    storage_account     = azurerm_storage_account.this.name
+    storage_account_key = azurerm_storage_account.this.primary_access_key
+  }
+
+  dependency_ids = {
+    cert-manager = module.cert-manager.id
```

```suggestion
    argocd       = module.argocd_bootstrap.id
    traefik      = module.traefik.id
    cert-manager = module.cert-manager.id
    loki-stack   = module.loki-stack.id
```

The same comment as in the final Argo CD module.